js/bars.js

```diff
@@ -1128,6 +1205,33 @@ historian.Bars.prototype.tooltipText_ = function(
   }
 
   formattedLines.push(cluster.clusteredCount + ' occurences');
+  if (series.source == historian.historianV2Logs.Sources.SENSORSERVICE_DUMP) {
+    var sensor;
+    historian.sensorsInfo.Sensors.forEach(function (sensorObj) {
+      // Accomodate the case where sensor number is 0, the protobuf
+      // message does not have the field for Number
```

Comment:
un-capitalize Number. The statement seems a bit incomplete.

Reply:
This code & the comment is repeated in lines 825-835. Is it possible to have only 1 occurrence of this code?